dg-commons/src/main/code/Graph/Node.scala

```diff
@@ -0,0 +1,205 @@
+package Graph
```

Comment:
I don't see the apache header on any of the files. Can you make sure that the checkstyle rules are being applied here ? That applies to all the files.